requirements.txt

```diff
@@ -0,0 +1,4 @@
+msprime
+scikit-allel
+snakemake
+parallel
```

Comment:
I remember @agladstein had some problems with this. Is this referring to GNU parallel or to a Python package called ``parallel``? The requirements.txt file should only refer to Python packages, it would cause all sorts of confusing otherwise.

Reply:
see https://github.com/andrewkern/analysis/issues/2
That's GNU parallel.

So far I changed the requirements.txt to 
```
setuptools
msprime
scikit-allel
snakemake
```
And then https://github.com/andrewkern/analysis/issues/2#issuecomment-470364921
But I still don't have the environment correctly set up and haven't gotten the pipeline to run successfully yet...

Reply:
My vote would be to not get too caught up in defining the environment just yet and get some code in place first.

Reply:
_I_ at least have to have the environment working to try it out 😆

Reply:
yes it's GNU parallel. Ideally I'm hoping to remove the use of `parallel` and switch all the computation over to cluster calls using `snakemake` but have this in for the meantime to flesh out a MWE. Sound ok for now? Once we merge we can remove it from the `requirements.txt` file.

Reply:
I would be better to remove it I think, as we know it shouldn't be in there. People will expect to use requirements.txt like ``pip install -r requirements.txt``, which this breaks.

I've no issue with using GNU parallel, it just doesn't belong in this file is all.

Reply:
done

Reply:
I think you might also need to add `setuptools` in the `requirements.txt

Reply:
we need that for the `stdpopsim` repo but not this one specifically (yet). can add it though.

Reply:
We usually assume ``setuptools`` these days I think.

Reply:
hmm... I think I couldn't get the msprime dependencies to build without it.

Reply:
We also need `matplotlib`

Reply:
> hmm... I think I couldn't get the msprime dependencies to build without it.

I'm happy to resolve this later if there's a problem (maybe create an issue?). I think we should merge what Andy has done here before trying to get the environment exactly right --- we're never going to get all the packages, and it's an easy one to follow up on afterwards with a PR.